README.md

```diff
@@ -61,6 +61,10 @@ There are two types of configuration that `wrangler` uses: global user and per p
     edit this file to add these values before you can publish.
 
     - `name`: This is the name of your project. It will be the name of your script.
+    - `type`: The type of project this is: can be:
```

Comment:
the double colon here is a bit weird. can we update t o remove th at

Reply:
Changed.